posthog-node/src/posthog-node.ts

```diff
@@ -203,6 +204,32 @@ export class PostHog implements PostHogNodeV1 {
     return response
   }
 
+  async getFeatureFlagPayload(key: string, distinctId: string, value: string | boolean, options?: {
```

Comment:
I'm thinking the API looks like this where the user can pass in the matched value and we just handle the lookup. alternatively, we can remove this argument and have this function handle/trigger a compute for the matched flag and then lookup too

Reply:
Only for the multivariate case I guess? As otherwise there's only one payload?

I think it makes sense to stick to the same interface as for js, so just key and distinctId. This seems to make it more error prone.